crates/egui/src/widgets/plot/mod.rs

```diff
@@ -1088,6 +1093,27 @@ impl PlotUi {
         &self.response
     }
 
+    /// Scale the plot bounds around a position in screen coordinates. Can be useful for
+    /// implementing alternative plot navigation methods.
+    /// The plot bounds are divided by `zoom_factor`, therefore:
+    /// `zoom_factor > 1.0` zooms in, i.e., reduces the visible range to show more detail.
+    /// `zoom_factor < 1.0` zooms out, i.e., increases the visible range to show more data.
+    pub fn zoom_bounds(&mut self, zoom_factor: Vec2, center: PlotPoint) {
+        self.bounds_modifications
+            .push(BoundsModification::Zoom(zoom_factor, center));
+    }
+
+    /// Scale the plot bounds around the hovered position, if any. Can be useful for implementing
+    /// alternative plot navigation methods.
+    /// The plot bounds are divided by `zoom_factor`, therefore:
+    /// `zoom_factor > 1.0` zooms in, i.e., reduces the visible range to show more detail.
+    /// `zoom_factor < 1.0` zooms out, i.e., increases the visible range to show more data.
```

Comment:
nit: let's describe them in number-line order

```suggestion
    /// `zoom_factor < 1.0` zooms out, i.e., increases the visible range to show more data.
    /// `zoom_factor > 1.0` zooms in, i.e., reduces the visible range to show more detail.
```